Source/EasyNetQ.ApprovalTests/EasyNetQ.approved.txt

```diff
@@ -1454,7 +1454,7 @@ namespace EasyNetQ.Internals
         public void Decrement() { }
         public void Dispose() { }
         public void Increment() { }
-        public void Wait() { }
+        public System.Threading.Tasks.Task WaitAsync() { }
```

Comment:
Can you add the namespaces to the usings?

Reply:
This is approval txt file generated by `PublicApiGenerator` package. Full names usage is by design.

Reply:
Ahh sorry I missed that 😞